recipes/antarna/meta.yaml

```diff
@@ -13,13 +13,15 @@ build:
 
 requirements:
   build:
-    - python
+    - python 2.7*
+    - argparse
     - numpy
     - uuid
     - networkx
     - scipy
   run:
-    - python
+    - python 2.7*
+    - argparse
```

Comment:
No, no ;) please merge the PR that I have created against your repo and revert this one ;)